website/docs/language/settings/backends/remote.html.md

```diff
@@ -8,17 +8,22 @@ description: |-
 
 # remote
 
-**Kind: Enhanced**
+-> **Note:** With the release of Terraform v1.1.0, we recommend using the Terraform Cloud's built-in
+[`cloud` integration](/docs/language/settings/terraform-cloud.html) instead of this backend.
+The `cloud` option includes an improved user experience and more features.
```

Comment:
Carrying over from the other PR - we don't want to have three notes in a row!
```suggestion
```

Reply:
I'd like to keep this here and make the next thing not a note. That way versioning recommendations stay in notes and are separated by some text.

Reply:
```suggestion
-> **Note:** The remote backend was introduced in Terraform v0.11.13 and Terraform Enterprise v201809-1. As of Terraform v1.1.0 and Terraform Enterprise v202201-1, **we recommend using the Terraform Cloud's built-in [`cloud` integration](/docs/language/settings/terraform-cloud.html)** instead of this backend. The `cloud` option includes an improved user experience and more features. 


```